test/CMakeLists.txt

```diff
@@ -434,6 +467,10 @@ _Block_release(void) { }\n")
             PROPERTY FOLDER "Tests/check-swift")
       endforeach()
     endforeach()
+
+    write_component_test_script("${test_bin_dir}" "irgen" "IRGen" )
```

Comment:
Please limit this to systems where these can actually work.  CMD is the only shell you can use on Windows :)

Reply:
Done, PTAL